modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -57,12 +58,16 @@
 					_this.setLiveCuepointsWatchDog();
 				}
 			});
+
+            this.baseThumbAssetUrl=null;
+            this.disableThumbnailAssetUrlFetching=mw.getConfig("EmbedPlayer.disableThumbnailAssetUrlFetching");
```

Comment:
rename disableThumbnailAssetUrlFetching to disableThumbnailAssetUrlFetchingOptimization